x/accesscontrol/types/access_operations.go

```diff
@@ -0,0 +1,63 @@
+package types
+
+import (
+	acltypes "github.com/cosmos/cosmos-sdk/types/accesscontrol"
+)
+
+type AccessOperationSet struct {
```

Comment:
should we also add a function to convert the map into a list of access operations? I don't think the order matters apart from the fact that the COMMIT access op should be last

Reply:
added